### PR TITLE
Add release notes for v4.0.17

### PR DESCRIPTION
**Pull request by andre-gh**

### CODE REVIEW

File: site/pages/docs/versions/dwnld-en.hbs [hunk 1]
```diff
@@ -31,11 +31,55 @@
 		</li>
```
Comment (LaurentGoderre) [inline site/pages/docs/versions/dwnld-en.hbs]: Please update the date modified

File: site/pages/docs/versions/dwnld-fr.hbs [hunk 2]
```diff
@@ -36,6 +36,51 @@
 	<h2 id="v40">Version 4.0</h2>
```
Comment (LaurentGoderre) [inline site/pages/docs/versions/dwnld-fr.hbs]: Please update the date modified

File: site/pages/docs/versions/v4.0/v4.0.17-en.hbs [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+	<li><strong>Release date:</strong> <time>2015-08-21</time></li>
+	<li><a href="../dwnld-en.html#v4017">Downloads</a></li>
+</ul>
+
+<span class="wb-prettify all-pre"></span>
+
+<section>
+	<h2>What's New</h2>
+	<ul>
+		<li>Datalist: Fix inconsistent matches between browsers</li>
```
Comment (LaurentGoderre) [inline site/pages/docs/versions/v4.0/v4.0.17-en.hbs]: For consistency with other releases, pleas bold the plugin names

**Comment by andre-gh** [comment]
Updated

**Comment by andre-gh** [comment]
Bolded plugin names

**Comment by LaurentGoderre** [comment]
Could you squash the commit now that it's ready?

**Comment by andre-gh** [comment]
Squished

**Comment by andre-gh** [comment]
Good to merge?

**Comment by LaurentGoderre** [comment]
It's missing a change I think

**Comment by LaurentGoderre** [comment]
Last files that needs to be changed are

https://github.com/wet-boew/wet-boew/blob/master/site/pages/docs/versions/v4.0/v4.0-changelog-en.hbs
https://github.com/wet-boew/wet-boew/blob/master/site/pages/docs/versions/v4.0/v4.0-changelog-fr.hbs

**Comment by andre-gh** [comment]
Fixed

**Comment by andre-gh** [comment]
Clears Travis, ok to merge?

**Pull request merged**